packages/rum/src/apm-base.js

```diff
@@ -135,15 +135,15 @@ class ApmBase {
       managed: true,
       canReuse: true
     })
-
     if (tr) {
       tr.addTask(PAGE_LOAD)
```

Comment:
we still have the task here since the page-load cannot be closed by any other detectfinish logic